tests/snuba/tasks/test_unmerge.py

```diff
@@ -320,7 +320,7 @@ def create_message_event(template, parameters, environment, release, fingerprint
 
         assert set(
             GroupHash.objects.filter(group_id=source.id).values_list("hash", flat=True)
-        ) == set([events.keys()[0], events.keys()[1]])
+        ) == set([list(events.keys())[0], list(events.keys())[1]])
```

Comment:
```suggestion
        ) == set(itertools.islice(events.keys(), 2))
```

See `islice` here: https://docs.python.org/2/library/itertools.html#itertools.islice